pages/staff/academic-staff/Thilina Gunarathne

```diff
@@ -0,0 +1,32 @@
+---
+layout: staffDetails
+permalink: /staff/academic/nadeesha-adikari/
+title: Mr. Thilina Gunarathne
+
+name_below_image: Mr. Thilina Gunarathne
+contact_number:
+email: thilinag@eng.pdn.ac.lk
+location: #
+
+text_below_name: Lecturer(Prob)
+
+url_image: images/staff/academic-staff/default.jpg
+url_website: #
+url_google_scholar: #
+url_orcid: #
+url_ad_scientific_index: #
+
+designation: Lecturer(Prob) 
+is_hod: false
+on_duty: false
+
+research_interests: [ 'Image Processing', 'Programming Languages', 'Software Engineering' ]
+
+---
+
+<i class="fas fa-user"></i>
+<a><b><u>BIO</u></b></a>
+
+<p>
+    Mr.Thilina Gunarathne  received B.Sc (Eng) in Computer Engineering from University of Vauniya, Sri Lanka in 2020  She joined the University of Peradeniya as a Lecturer (on contract) 2020.
```

Comment:
Content-related review is required on the following:
1. `She` joined the 
2. Lecturer (on contract) `2020`.